### PR TITLE
Adding yaml for Ford Data

### PR DESCRIPTION
**Pull request by agarwalsid**
*Issue #, if available:*

*Description of changes:*

Adding ford-multi-av-seasonal yaml file for dataset

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/ford-multi-av-seasonal.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  -robotics
+  -sensors
+  -localization
+  -mapping
+  -aws-pds 
+License: This data is intended for non-commercial academic use only. It is licensed under the Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License.
+Resources:
+  - Description:
+    ARN:arn:aws:s3:::ford-multi-av-seasonal
+    Region:US West
```
Comment (jflasher) [inline datasets/ford-multi-av-seasonal.yaml]: Please change to a format like us-west-2.

File: datasets/ford-multi-av-seasonal.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Contact: https://avdata.ford.com/
+ManagedBy: Ford Motor Company
+UpdateFrequency: New data will be added until the entire dataset is released online.
+Tags:
+  -autonomous vehicles
```
Comment (jflasher) [inline datasets/ford-multi-av-seasonal.yaml]: Please use tags from https://github.com/awslabs/open-data-registry/blob/master/tags.yaml.

File: datasets/ford-multi-av-seasonal.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  -mapping
+  -aws-pds 
+License: This data is intended for non-commercial academic use only. It is licensed under the Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License.
+Resources:
+  - Description:
+    ARN:arn:aws:s3:::ford-multi-av-seasonal
+    Region:US West
+    Type:S3 Bucket
+DataAtWork:
+  Tutorials:
```
Comment (jflasher) [inline datasets/ford-multi-av-seasonal.yaml]: Please include AuthorName and Title.

File: datasets/ford-multi-av-seasonal.yaml (hunk 1 above)
Comment (agarwalsid) [reply datasets/ford-multi-av-seasonal.yaml]: Done, Thanks!

File: datasets/ford-multi-av-seasonal.yaml (hunk 2 above)
Comment (agarwalsid) [reply datasets/ford-multi-av-seasonal.yaml]: Done, Thanks!

File: datasets/ford-multi-av-seasonal.yaml (hunk 3 above)
Comment (agarwalsid) [reply datasets/ford-multi-av-seasonal.yaml]: Done, Thanks!

**Pull request merged**